tests/x509/limbo/test_limbo.py

```diff
@@ -25,10 +25,6 @@
     # Our support for custom EKUs is limited, and we (like most impls.) don't
     # handle all EKU conditions under CABF.
     "pedantic-webpki-eku",
-    # Contains tests that fail as a matter of strict CA/BF policy, but
```

Comment:
@woodruffw Is there anything stopping us from running this? Looks to be passing fine on my end.
We need these tests for coverage (in particular, the ones testing for forbidden signature algorithms).

Reply:
Hmm, one of the testcases tagged with this was failing before, but maybe we've fixed it.